jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/util/HttpRequestUtil.java

```diff
@@ -86,6 +86,16 @@ public static String extractAeroGearSenderInformation(final HttpServletRequest r
     }
 
     /**
+     * Extract the header that identifies the push message.
+     *
+     * @param request
+     * @return
```

Comment:
please do full JavaDoc - on Java8 incomplete JavaDoc is crying like a baby. see also https://github.com/aerogear/aerogear-unifiedpush-server/pull/538